visualization/include/pcl/visualization/impl/pcl_visualizer.hpp

```diff
@@ -1696,30 +1696,30 @@ pcl::visualization::PCLVisualizer::addPolygonMesh (
   {
     // Create polys from polyMesh.polygons
     vtkSmartPointer<vtkCellArray> cell_array = vtkSmartPointer<vtkCellArray>::New ();
-    vtkIdType *cell = cell_array->WritePointer (vertices.size (), vertices.size () * (max_size_of_polygon + 1));
+    // vtkIdType *cell = cell_array->WritePointer (vertices.size (), vertices.size () * (max_size_of_polygon + 1));
```

Comment:
Delete all commented code you've introduced. Deleting those lines is perfectly ok. Apply this logic to the rest of the changes you did in this file.

Do you see the code from lines 1690-1693? Since `max_size_of_polygon` is no longer used, that section is also no longer required.